wiseService/vueapp/src/components/Config.vue

```diff
@@ -154,21 +162,107 @@
               </a>
             </template>
           </p>
-          <!-- text area input for non json -->
-          <b-form-textarea
-            v-if="!currJSONFile"
-            v-model="currFile"
-            rows="18"
-          />
+          <!-- text area input for non json/csv -->
+          <template
+            v-if="(!currJSONFile && currConfig[selectedSourceKey].format === 'tagger') || rawCSV">
+            <b-form-textarea
```

Comment:
If the format is CSV and the user is editing in the text area, should I clarify the CSV format? Maybe say something like:

> Rows are delimited by newlines. Cells are delimited by commas. Comments are delimited by # and should be an entire row by itself.